Scripts/Linux/Scalar_Clone.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
Is this script necessary? Or is it only because we can't get a binary named `scalar` on the `PATH` (yet)?

Reply:
I notice there is a similar script in `Scripts/Mac`, but it's probably not necessary, either.

Reply:
I just copied this one from the Mac side of things.  No idea what it does yet.  :-)

Reply:
The Mac one is part of a set of three, `GVFS_Mount.sh`, `GVFS_Unmount.sh`, and `GVFS_Clone.sh`, which date from some of the earliest public bulk commits in that project.  Originally all three worked on a common `~/GVFSTest` [mount point](https://github.com/microsoft/VFSForGit/blob/0eeb14b5f98adedecc332410a717dc2e9968ab6e/Scripts/Mac/GVFS_Mount.sh#L10) and were, I believe, just handy utilities for doing manual tests and simplifying the mount/clone/unmount steps.

The mount/unmount ones were copied to Scalar but then removed in #216, leaving just `Scalar_Clone.sh`.  If no one is using it, I'd agree it could just be removed.